boot/seal.go

```diff
@@ -442,11 +429,11 @@ func resealKeyToModeenvImpl(rootdir string, modeenv *Modeenv, expectReseal bool,
 	return resealKeyToModeenvForMethod(method, rootdir, modeenv, expectReseal)
 }
 
-type resealKeyForBootChainsParams struct {
-	runModeBootChains           []bootChain
-	recoveryBootChainsForRunKey []bootChain
-	recoveryBootChains          []bootChain
-	roleToBlName                map[bootloader.Role]string
+type ResealKeyForBootChainsParams struct {
+	RunModeBootChains           []bootChain
```

Comment:
do you need s/bootChain/BootChain/ too?

Reply:
The capitalization of some of those names are only temporary until everything is moved to `overlord/fdestate/backend`. So I will see later what I need to make public. But maybe `bootChain` will move. Anyway, I do the minimum here.

Reply:
Done along with @pedronis related comment.